packages/calcite-components/src/components/action/action.scss

```diff
@@ -4,6 +4,9 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-action-indicator-color: Specifies the color of the component's indicator.
+ * @prop --calcite-action-loader-color-oneThird: Specifies the starting color of the loader
```

Comment:
Its weird to have camel casing in the name here. Should we have `one-third`? or something similar to keep hyphen cased?